modules/RTC/TPCUtils.js

```diff
@@ -232,8 +232,8 @@ export class TPCUtils {
             this.pc.peerconnection.addTrack(track);
         }
 
-        // Construct the simulcast stream constraints for the newly added track.
-        if (localTrack.isVideoTrack() && localTrack.videoType === VideoType.CAMERA && this.pc.isSimulcastOn()) {
+        // Construct the stream constraints for the newly added track.
+        if (localTrack.isVideoTrack() && localTrack.videoType === VideoType.CAMERA) {
```

Comment:
I am constructing the stream constraints for both simulcast and unicast cases so that the bitrates values can be used for the unicast case when the stream is downsampled to get a lower resolution. I will rename the simulcastEncodings in TPCUtils in a separate commit so that is more clear.